.travis.yml

```diff
@@ -38,6 +38,7 @@ before_install:
 
 install:
   - npm ci
+  - cd node_modules/fxa-geodb/db/ && wget https://s3-us-west-2.amazonaws.com/fxa-dev-bucket/notes/geodb/sept27/cities-db.mmdb && cd ../../.. 
```

Comment:
Might be worthwhile to add a comment here why we are downloading from S3.

Reply:
added